kubecost/values.yaml

```
@@ -138,6 +138,36 @@ global:
         #   - property: "service"
         #     value: "service1" # corresponds to a value to filter cloud cost aggregate by service data on.
 
+  ## Kubecost Budget and Budget Actions
+  ## Ref: https://www.ibm.com/docs/en/kubecost/self-hosted/3.x?topic=ui-budgets
+  ##
+  budget:
```

```suggestion
  budgets:
```
Needs to be plural. May need to update elsewhere.